docs/client-sdk/javascript/tutorials/use-content-types.mdx

```diff
@@ -61,58 +61,148 @@ To do this, pass the `CompositeCodec` content type as an option to the `Client.c
 
 :::
 
-If another app wants to display your custom content type, they must implement your custom content type in their code exactly as it's defined in your code.
+## Send a custom content type
 
-:::caution
+Once you’ve built your custom content type, you can immediately start using it to tag and send messages. This task uses a fictional `AudioCodec` custom content type to help illustrate the steps. 
 
-Do not use `xmtp.org` as the `authorityId` for a custom content type that you create for your app. `xmtp.org` is reserved for use as the `authorityId` for standard content types only.
+1. Register your AudioCodec custom content type with the message API client. For example:
+
+  ```javascript
+  import { Conversation, ContentTypeId } from '@xmtp/xmtp-js';
+
+  const ContentTypeVoiceKey = new ContentTypeId({
+    authorityId: 'xmtp.test',
+    typeId: 'audio-key',
+    versionMajor: 1,
+    versionMinor: 0
+  });
+  ```
+2. Send a message using the `send` method with `audioRecording` as the content (in byteArrays). 
+
+  For example, you can build the `AudioCodec` to encode `audioData` as a binary representation of the audio file. 
+
+  In this code sample, the message is tagged and sent with two options: `contentType` and `contentFallback`. The `contentType` option is set to `ContentTypeVoiceKey`, which enables the message API client to know to use the `AudioCodec` to encode and decode the content. The `contentFallback` option is set to 'This is an audio recording' to enable an app to display descriptive text if it is unable to handle the custom content type.
+
+  ```javascript
+  const audioRecording = 'xxxxxxx'; // audio recording in byteArrays
+  conversation.send(audioRecording, {
+    contentType: ContentTypeVoiceKey,
+    contentFallback: 'This is an audio recording'
+  })
+  ```
+
+  As a best practice, use the `contentFallback` option to provide an alt text-like plain text description that will display in place of the original content in the message if an app is unable to handle your custom (non-standard) content type.
+
+:::info
+
+Your custom content type WILL NOT automatically be supported by other apps and will display fallback text in them instead.
 
 :::
 
-To learn more about `authorityId` and other parameters in a content type identifier, see [Content Type Identifier and Parameters](https://github.com/xmtp/XIPs/blob/ae6fc638332f57f918d82a096f69b1e79df0bd0a/XIPs/xip-5-message-content-types.md#content-type-identifier-and-parameters).
+If another app wants to display your custom content type, they must implement your custom content type in their code exactly as it's defined in your code.
 
-**To send a custom content type:**
+  If developers take interest in your custom content type, consider proposing it as a standard through the [XIP process](https://github.com/xmtp/XIPs#readme).
 
-1. Specify a custom content type to enable during client initialization.
+  :::info
```

Comment:
I think lines 164-170 are repeats of lines 154-160?

Reply:
you're absolutely right! don't know how that happened! Thanks!